### PR TITLE
ci(rc): enable instrumented test

### PR DESCRIPTION
**Pull request by trOnk12**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

This is a cherry-pick of https://github.com/wireapp/wire-android-reloaded/pull/1569

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1721/1/) **failed**.

File: .github/workflows/gradle-run-ui-tests.yml [hunk 1]
```diff
@@ -12,11 +12,10 @@ concurrency:
 
 jobs:
   ui-tests:
-    if: ${{ contains(github.head_ref, 'qa/') || contains(github.ref_name, 'qa/')}}  # disable for as they are not yet completed
     runs-on: buildjet-8vcpu-ubuntu-2204
     strategy:
       matrix:
-        api-level: [30]
+        api-level: [29]
```
Comment (tmspzz) [inline .github/workflows/gradle-run-ui-tests.yml:18]:
```suggestion
        api-level: [30]
```

**Review by tmspzz — APPROVED** [review]

File: .github/workflows/gradle-run-ui-tests.yml (hunk 1 above)
Comment (trOnk12) [reply .github/workflows/gradle-run-ui-tests.yml:18]: api level 30 is acting weird I advice to keep it on 29 as I have tested on both apis and 30 time out 7/10 times where 29 never did

**Review by MohamadJaara — APPROVED** [review]

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1721/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.1(6353839)(6353839)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.1(6353839)(6353839)-staging-compat.apk)

**Pull request merged**